src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,29 @@
+.catalog {
+  --column-count: 1;
+
+  box-sizing: border-box;
+  padding: 50px 40px;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(var(--column-count), $card-width);
```

Comment:
Ensure that the SCSS variable `$card-width` is defined elsewhere in your styles. This variable is crucial for the grid layout to function correctly.